test_cli.py

```diff
@@ -923,6 +925,59 @@ def test_worksheet_tags(ctx):
     check_contains(r'Tags:\s+###', _run_command([cl, 'ls', '-w', wuuid]))
 
 
+@TestModule.register('uls')
+def test(ctx):
+    prev_time = datetime.now().isoformat()
+    # Create new user
+    user = User(
+        {
+            "user_id": 1,
+            "user_name": "test",
+            "email": "test@test.com",
+            "notifications": NOTIFICATIONS_IMPORTANT,
+            "last_login": datetime.datetime.now(),
+            "is_active": True,
+            "first_name": None,
+            "last_name": None,
+            "date_joined": datetime.datetime.now(),
+            "has_access": True,
+            "is_verified": True,
+            "is_superuser": True,
+            "password": "",
+            "time_quota": 10,
+            "parallel_run_quota": 0,
+            "time_used": 1,
+            "disk_quota": 10,
+            "disk_used": 1,
+            "affiliation": None,
+            "url": None,
+        }
+    )
+
+    # check .joined_after
+    check_contains(
+        'test', _run_command([cl, 'uls', '.joined_after=' + prev_time, '-f', 'user_name'])
+    )
+
+    # check .count
+    check_equals('1', _run_command([cl, 'uls', 'joined_after=' + prev_time, '.count']))
+
+    # check .active_after
+    check_contains(
+        'test', _run_command([cl, 'uls', '.active_after=' + prev_time, '-f', 'user_name'])
+    )
+
+    # check .disk_used_more_than
+    check_contains(
+        ['test'], _run_command([cl, 'uls', '.disk_used_more_than=' + '10%', '-f', 'user_name'])
+    )
+
+    # check .time_used_more_than
+    check_contains(
+        ['test'], _run_command([cl, 'uls', '.time_used_more_than=' + '10%', '-f', 'user_name'])
+    )
```

Comment:
Can you make sure that these tests test _all_ possible filters (such as disk_quota, etc.)?